### ISSUE TITLE
doc: Add proper Accessibility Guidelines to Documentation Style Guide

### ISSUE THREAD

**Issue by eeshaanSA** [kubestellar/kubestellar#3160]
### Describe the issue

The current KubeStellar Documentation Style Guide lacks clear instructions about accessibility standards (we currently have a link to w3 standards under alt-texts heading). Accessibility ensures documentation is inclusive and usable by contributors around the world.

Addressing this issue would mean adding onto the current abstract section of accessibility, which will in turn (in the near future) ensure the KS Documentation remains inclusive and usable by contributors from diverse backgrounds, including those with disabilities. 

Having this crucial detail in the Style Guide willl definitely increase community involvement, making contributions easier and more welcoming.

### Want to contribute?

- [x] I would like to work on this issue.

**Comment by eeshaanSA** [comment]
cc: @clubanderson @KPRoche 

Post your approval, let us add this under the Epic issue of improving the Docs Style Guide.

**Comment by KPRoche** [comment]
It's a sign of how long I've been doing webpages (all the way back to html 2) that I linked to the w3c accessibility standards, @MikeSpreitzer quite rightly pointed that they won't mean much to contributors working in markdown, and also that we probably don't need to write Yet Another Manual On MarkDown, but should maybe include some basic guidelines, (for instance, actually show the syntax to include alt-text in MD, which I just had to look up) and link to established sites documenting more detailed standards.

**Comment by eeshaanSA** [comment]
Absolutely, you are right. 9 out of 10 contributors contributing to the docs (or markdown) would not care to check a style guide, but adding some details also means that if someone does actually write something which isn't quite as accessible, we will always have something in our docs to point them to. 

W3C is no doubt good, but what we can also add is as you said, maybe some snippets, maybe some examples, and also (in addition to W3C), some better links like WCAG (Web content accessibility guidelines), some good articles (from good sources of course), on how to reduce bias while writing documentation. 

Thoughts?

**Comment by KPRoche** [comment]
> W3C is no doubt good, but what we can also add is as you said, maybe some snippets, maybe some examples, and also (in addition to W3C), some better links like WCAG (Web content accessibility guidelines), some good articles (from good sources of course), on how to reduce bias while writing documentation.
> 
> Thoughts?

You may be amused to know that the first link returned by a Google search for WCAG is in fact a W3C page (the parent page to the page I linked in the base Style Guide, in fact!)

One of the first links returned by "WCAG and MarkDown" is https://www.smashingmagazine.com/2021/09/improving-accessibility-of-markdown/

**Comment by Deathnote45** [comment]
hi @eeshaanSA can I contribute on this issue?

**Comment by KPRoche** [comment]
Hi @Deathnote45 I expect this issue is going to result in a number of PRs. more contributors are helpful!
Be sure to check out the parent Epic #3130 if  you have not already!

**Comment by eeshaanSA** [comment]
> > W3C is no doubt good, but what we can also add is as you said, maybe some snippets, maybe some examples, and also (in addition to W3C), some better links like WCAG (Web content accessibility guidelines), some good articles (from good sources of course), on how to reduce bias while writing documentation.
> > Thoughts?
> 
> You may be amused to know that the first link returned by a Google search for WCAG is in fact a W3C page (the parent page to the page I linked in the base Style Guide, in fact!)
> 
> One of the first links returned by "WCAG and MarkDown" is https://www.smashingmagazine.com/2021/09/improving-accessibility-of-markdown/

No way, I have been advocating for WCAG 2.x for long enough now, and I never realized (subconsciously) that the first link is almost always a W3C link. This is interesting. I might have to think more about and carefully include the resources I am planning to include. Thanks for this precious info, @KPRoche 👍

**Comment by eeshaanSA** [comment]
Dear @Deathnote45 and @adity1raut, thank you for your interest in solving this issue. This issue is part of a more bigger, larger, [EPIC] issue #3130, where your contributions would be very valuable!

**Comment by KPRoche** [comment]
> Dear [@Deathnote45](https://github.com/Deathnote45) and [@adity1raut](https://github.com/adity1raut), thank you for your interest in solving this issue. This issue is part of a more bigger, larger, [EPIC] issue [#3130](https://github.com/kubestellar/kubestellar/issues/3130), where your contributions would be very valuable!

As a note, the path I'm encouraging to add issues to the Epic is
- create an issue on the revision you are suggesting. 
- add a comment tagging me (or another maintainer) in that issue asking it be linked to the epic (I try to jump on these ASAP)
- add a short comment to the epic describing and pointing at your sub-issue
- when it's time for a PR, link it to your sub-issue so it closes that rather than the main (Epic) issue.

I'm also trying to encourage smaller, incremental PRs rather than giant ones so merging is easier and we can evolve rather than replace the docs.
